### PR TITLE
Move to mytoken for deployment

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Moving to an alternative way of handling auth for deployment:
1. Use of oidc clientid/refresh token  support will be soon removed from fedcloudclient
2. fedcloudclient does not support the newer client https://aai.egi.eu/fedcloud2 
3. mytoken seems to work and reasonably secure

---

**Related issue :**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ on:
     branches:
       - main
   pull_request:
-    paths:
-      - "deploy/**"
+  #  paths:
```
Comment (gwarf) [inline .github/workflows/deploy.yml:9]: What's the purpose of this?

Comment (enolfc) [reply .github/workflows/deploy.yml:9]: I'm trying to understand why the deployment is failing. Forcing the execution of this workflow always for testing for the time being. Should be put back as before as soon as it starts working

Comment (enolfc) [reply .github/workflows/deploy.yml]: Back as usual. Should be working now

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**